HelpSource/Help.schelp

```diff
@@ -2,38 +2,328 @@ TITLE::Help
 summary::Documentation home
 categories::Help
 
-SuperCollider is an audio server, programming language, and IDE for sound synthesis and algorithmic composition.
-
 NOTE:: link::Guides/News-3_13##News in SuperCollider version 3.13:: ::
 
-SECTION::Search and browse
+
+SECTION:: An introductory overview
+
+emphasis::SuperCollider:: is a cross-platform environment for audio synthesis and algorithmic composition used by musicians, artists and researchers working with sound. It can be installed and run on link::https://supercollider.github.io/downloads.html#linux#Linux::,
```

Comment:
Maybe just link to the downloads page by the word "installed" rather than linking to each individual OS download.

Reply:
Done, thanks!